### PR TITLE
Include second solo test execution in total test count

### LINKED ISSUE #1017: [Test issue] The tests don't correctly reflect the number of tests left
It repeatedly says we are at 124/124 when running, https://github.com/valkey-io/valkey/actions/runs/10805625870/job/29973022674?pr=1016#step:4:5651. I guess you can replicate this with just `make test`. Not sure what is causing it.

```
[124/124 done]: integration/replication-psync (324 seconds)
Testing solo test
=== () Starting server 127.0.0.1:28738 ok
[ignore]: XADD one huge field: large memory flag not provided
=== () Starting server 127.0.0.1:28739 ok
[ignore]: XADD one huge field - 1: large memory flag not provided
=== () Starting server 127.0.0.1:28740 ok
[ignore]: several XADD big fields: large memory flag not provided
=== () Starting server 127.0.0.1:28741 ok
[ignore]: single XADD big fields: large memory flag not provided
=== () Starting server 127.0.0.1:28742 ok
[ignore]: hash with many big fields: large memory flag not provided
=== () Starting server 127.0.0.1:28743 ok
[ignore]: hash with one huge field: large memory flag not provided
[124/124 done]: violations (1 seconds)
Testing solo test
=== (bitops) Starting server 127.0.0.1:28744 ok
[ignore]: BIT pos larger than UINT_MAX: large memory flag not provided
[ignore]: SETBIT values larger than UINT32_MAX and lzf_compress/lzf_decompress correctly: large memory flag not provided
[124/124 done]: bitops-large-memory (1 seconds)
```

### PR DESCRIPTION
**Pull request by Shivshankar-Reddy**
This change excludes the solo tests from completed tests count. Fixes: https://github.com/valkey-io/valkey/issues/1017
Test results after the change:
image

### CODE REVIEW

**Comment by hwware** [comment]
@Shivshankar-Reddy Could you pls post the test result here? Then we can see the test count number changes. Thanks

**Comment by Shivshankar-Reddy** [comment]
> @Shivshankar-Reddy Could you pls post the test result here? Then we can see the test count number changes. Thanks

Updated the results in top comment. Thanks!

**Review by hwware — APPROVED** [review]

File: tests/test_helper.tcl [hunk 1]
```diff
@@ -370,7 +371,13 @@ proc read_from_test_client fd {
         set all_tests_count [llength $::all_tests]
         set running_tests_count [expr {[llength $::active_clients]-1}]
         set completed_tests_count [expr {$::next_test-$running_tests_count}]
-        puts "\[$completed_tests_count/$all_tests_count [colorstr yellow $status]\]: $data ($elapsed seconds)"
+        # Solo tests are excluded from the completed_tests_count. Ref: #valkey/issues/1017
```
Comment (enjoy-binbin) [inline tests/test_helper.tcl]: i dont think we need this comment, we have a git blame log so we can save the issue ref.

**Review by enjoy-binbin — COMMENTED** [review]
Can we find some ways that also can show the progress of the solo tests?

**Comment by madolson** [comment]
> Can we find some ways that also can show the progress of the solo tests?

I think we should be able to use the run_solo queue as an indicator of the pending items left to address binbins comment.

**Review by madolson — APPROVED** [review]
```
[127/128 done]: list-large-memory (0 seconds)
Testing solo test
=== () Starting server 127.0.0.1:27266 ok
[ignore]: SADD, SCARD, SISMEMBER - large data: large memory flag not provided
[128/128 done]: set-large-memory (0 seconds)
```

Yeah, does about what we want.

**Pull request merged**